### PR TITLE
Diffusion - Extras

### PR DESCRIPTION
**Pull request by mattmdjaga**

### CODE REVIEW

File: chapters/en/Unit 5 - Generative Models/Diffusion models/Simple explanation and showcase of external resources.mdx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Extras
+
+## Personalisation
+
+Although diffusion models and GANs are able to create many new unique items, they aren't always able to create the specifc thing you need. Hence, you're left with having to fine tune a model, which usually requires a lot of data and compute. However, there are some techniques that can be used to personalise a model with just a few examples.
```
Comment (charchit7) [inline chapters/en/Unit 5 - Generative Models/Diffusion models/Simple explanation and showcase of external resources.mdx]: Can we add here 1 more line on about that : GANs producing high-fidelity, low-diversity samples. Diffusion is on he other hand? WDYT?

**Comment by mattmdjaga** [comment]
@hwaseem04 we shouldn't add images like this. To add images :
Request to join the https://huggingface.co/hf-vision organization.
Upload an image to https://huggingface.co/datasets/hf-vision/course-assets, e.g. via the web UI.
Get the URL (e.g. https://huggingface.co/datasets/huggingface-course/audio-course-images/blob/main/all_models.png) right click to "Download" button and copy the link.
Use that in standard markdown like image

Also use mdx not md for the file. Think that it's just an extension on md that lets you do things like integrate gradio demos.

File: chapters/en/Unit 5 - Generative Models/Diffusion models/Introduction - Diffusions.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+These generative models raised the bar to a new level in the area
+of generative modeling, particularly referring to models such
+as Imagen and Latent Diffusion Models (LDMs). For instance consider the below images generated via such models
+
+![Example images generated using diffusion models](files/diffusion-eg.png)
+
+GANs were considered by many as state-of-the-art generative models in terms of the quality of the generated samples, before the recent rise of diffusion models. GANs are also known
+as being difficult to train due to their adversarial objective,
+and often suffer from mode collapse ( where the generator fails to capture the entire distribution of the training data, resulting in repetitive or limited samples). In contrast, diffusion models have a stable training process and provide more diversity
+because they are likelihood-based.
```
Comment (charchit7) [inline chapters/en/Unit 5 - Generative Models/Diffusion models/Introduction - Diffusions.md]: Can we add here 1 more line on about that : GANs producing high-fidelity, low-diversity samples. Diffusion is on he other hand? WDYT?

**Review by charchit7 — APPROVED** [review]

File: chapters/en/Unit 5 - Generative Models/Diffusion models/Simple explanation and showcase of external resources.mdx (hunk 1 above)
Comment (mattmdjaga) [reply chapters/en/Unit 5 - Generative Models/Diffusion models/Simple explanation and showcase of external resources.mdx]: Don't think it's necessary in this section, I'm just meant to show external tools for diffusion models here. But it's a good thing to mention in one of the previous sections where we talk about GANs vs Diffusion models.

Comment (charchit7) [reply chapters/en/Unit 5 - Generative Models/Diffusion models/Simple explanation and showcase of external resources.mdx]: Yup, closing. Good PR!

**Pull request merged**